lib/daru/view/adapters/highcharts.rb

```diff
@@ -70,6 +70,21 @@ def export_html_file(plot, path='./plot.html')
           File.write(path, str)
         end
 
+        # Expoting in web frameforks is completely offline. In IRuby notebook,
+        #   offline-export supports only the exporting to png, jpeg and svg format.
+        #   Export to PDF is not working (not even through the exporting button in
+        #   highchart). So, online exporting is done in IRuby notebook. There is a
```

Comment:
We can open the issue to track it, right?

Reply:
I'll open it after this PR gets merged.